testssl.sh

```diff
@@ -2624,7 +2637,7 @@ test_just_one(){
                     ciphers_found[i]=true
                     if [[ ${kx[i]} == "Kx=ECDH" ]] || [[ ${kx[i]} == "Kx=DH" ]] || [[ ${kx[i]} == "Kx=EDH" ]]; then
                          dhlen=$(read_dhbits_from_file "$TMPFILE" quiet)
-                         kx[i]="${kx[i]} $dhlen"
+                         kx[i]="${kx[i]}($dhlen)"
```

Comment:
Nah, that looks odd for everything else than CSV output. Also the round brackets are colorized.